[Samples] README for add/remove operators (#127)
sendbird/sendbird-chat-sample-android: groupchannel-add-remove-operators/README.md

# Add or remove operators in a group channel

> This sample app demonstrates how to add and remove operators in a group channel using Sendbird Chat SDK for Android.

Author: ```suggestion
This sample with UI components demonstrates how to register or remove operators in a group channel on Sendbird Chat SDK for Android. Operators are users who can moderate channels by muting or banning users as well as freezing channels. To learn more, see our [documentation on operators](https://sendbird.com/docs/chat/v4/android/channel/managing-operators/register-and-remove-operators).
```

Reviewer: LGTM!

> ## Requirements

Author: ```suggestion
## Prerequisites
```

    + gradle: 7.5
    + android gradle plugin: 7.4.2

## How it works

> `ChatMemberListActivity.kt` is called when the user clicks the member list button in the `GroupChannelChatActivity.kt`.

Author: ```suggestion
When a user clicks the member list button in `GroupChannelChatActivity.kt`, the `changeMemberOperatorStatus()` method and [`ChatMemberListActivity.kt`](https://github.com/sendbird/sendbird-chat-sample-android/blob/02514ee6e46e598cd81f285b75174761b6b1c5d7/groupchannel-add-remove-operators/app/src/main/java/com/sendbird/chat/sample/groupchannel/addremoveoperators/user/ChatMemberListActivity.kt#L35) are called. 

The `changeMemberOperatorStatus()` method creates an adapter and change a user's role into an operator. If the current user wasn't an operator, their status changes to an operator. If the user was already an operator, their status remains the same. After a user's role is changed, the `refreshMembers()` method is called to refresh the member list.

As for the kit, `initRecyclerView()` is the first method to be called and the method sets the adapter to `RecyclerView`. 
```

Author: @sf-luke-cha adapter 에 대해 여기 적어주신 내용과 코드 첫번째 코멘트에 적어주신 내용이 달라 정확한 정보가 궁금합니다. 또한 아래 순서에 의해 코드가 작동하는 것이 맞는지요?
---
유저가 멤버 리스트 버튼 클릭 in GroupChannelChatActivity 킷 -> ChatMemberListActivity 킷의 initRecyclerView 메소드와 changeMemberOperatorStatus() 호출됨 & 어댑터 생성함(?) -> initRecylerView 메소드가 어댑터를 RecyclerView로 설정.
---

여기서 changeMemberOperatorStatus()는 호출될 때 어떻게 어댑터를 생성한 것인가요 (아래는 왜 initRecycleView 메소드 호출 이후 어댑터를 생성하라고 적혀있나요)?? 어댑터가 정확히 무엇인가요? recycler view는 또 어떤 설정인가요?

Reviewer: 제가 잘못 적은것 같습니다. changeMemberOperatorStatus()는 adapter에서 click이 수행될때 호출이 되도록 adapter를 생성할때  listener interface의 구현체를 정의합니다.

```
adapter = ChatMemberListAdapter { member, position -> changeMemberOperatorStatus(member, position) }
```
